### DISCUSSION TITLE
Decorative sprites don't rotate correctly with holdable items

### DISCUSSION THREAD

**Discussion by Aaron100-prog** [FakeFishGames/Barotrauma#15308] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

The decorative sprite component rotates weirdly with holdable items.

Example Artifact Holder Item:
![Bug Artifact Holder Decorative Sprite Offset](https://github.com/user-attachments/assets/36413d1b-4821-40c1-bca0-0109a38bd248)

### Reproduction steps

1. "spawnitem artifacttransportcase inventory"
2. Equip artifacttransportcase
3. (Go outside the sub and swim around)
4. See that the decorative sprite rotates weirdly with the holdable item

Happens to every holdable item that has decorative sprites attached and uses the offset attribute.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! This has been fixed in our private repo now, fix coming up in the next unstable build and the upcoming release.
